lntest/node.go

```diff
@@ -266,6 +268,8 @@ func newNode(cfg nodeConfig) (*HarnessNode, error) {
 	cfg.P2PPort, cfg.RPCPort, cfg.RESTPort = generateListeningPorts()
 
 	nodeNum := numActiveNodes
+	cfg.ProfilePort = 9000 + nodeNum
```

Comment:
Nit: include in `generateListeningPorts`?

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2957/commits/98712a1fc88ac8de4bf0ceab54c3da44a26d2f2a